sdk/communication/azure-communication-sms/tests/test_sms_client_e2e.py

```diff
@@ -44,3 +52,21 @@ def test_send_sms(self):
             send_sms_options=SendSmsOptions(enable_delivery_report=True))  # optional property
 
         assert sms_response.message_id is not None
+
+    @pytest.mark.live_test_only
+    def test_send_sms_from_managed_identity(self):
+        endpoint, access_key = parse_connection_str(self.connection_str)
+        from devtools_testutils import is_live
+        if not is_live():
```

Comment:
in earlier PR, I think there is an util function to create FakeTokenCredential or DefaultAzureCredential. I think we can leverage that function here.